src/client/components/SentReviewsComponent/SentReviewsComponent.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './SentReviewsComponent.styles.css';
+import unknownUser from '../../assets/images/portraits/unknown-black.png';
+import { RatingStars } from '../RatingStars/RatingStars';
+import { ReviewPost } from '../ReviewPost/ReviewPost';
+
+export const SentReviewsComponent = ({ reviews }) => {
+  return (
+    <div className="collection-reviews">
+      {reviews.map((review) => (
+        <div key={review.id} className="sent-review">
+          <div className="profile-picture">
+            <div className="circle-mask">
+              <img
+                src={review.cv.author.profileImageUrl || unknownUser}
+                alt="Portrait of user"
+                className="user-photo"
+              />
+            </div>
+          </div>
+          <div className="container-reviewee">
+            <div className="content">
+              <div className="reviewee-name-date">
+                <span className="reviewer-name">
+                  {review.cv.author.fullName}
+                </span>
+                <span className="review-date">
+                  {review.cv.createdDate.toLocaleString()}
+                </span>
+              </div>
+              <div className="stars-rating">
+                <RatingStars averageStars={review.cv.averageStars} />
+              </div>
+              <a
+                href={review.cv.fileUrl}
+                target="_blank"
+                rel="noopener noreferrer"
+                className="button button-orange"
+              >
+                See resume
+              </a>
+            </div>
+          </div>
+          <hr />
```

Comment:
Try to see if you can achieve this through css (it's not a blocker for this pr), just good practice.

Reply:
I sometimes just create something like `<div class="divider">` or something similar.

Reply:
Updated the files, just let me know if you catch some other improvement to make :) Thanks for the comment @pdsorensen!

Reply:
oh! and for the tip for dealing with lines